src/styles/index.scss

```diff
@@ -1,3 +1,203 @@
+$star-number: 5;
+$color-minor: #616070;
+$color-accent: #00acdc;
+
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+
+  background-color: #fff;
+  color: #060b35;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  box-sizing: border-box;
+
+  width: 200px;
+  height: 408px;
+
+  position: relative;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  font-weight: 400;
+  letter-spacing: 0;
+
+  &__image-container {
+    position: absolute;
```

Comment:
We don't need absolute positioning in the card and its content. Avoid using it everywhere. Consider trying to style the card within the normal flow of the document https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Normal_Flow